### ISSUE TITLE
Form elements should have a visible label

### ISSUE THREAD

**Issue by yusufttur** [shopware/platform#8430]
### Shopware Version

6.7.0.0

### Affected area / extension

Extension:B2B-Suite

### Actual behaviour

Ensure that every form element has a visible label and is not solely labeled using hidden labels, or the title or aria-describedby attributes
**How to fix:** https://dequeuniversity.com/rules/axe/4.10/label-title-only?application=AxeChrome

![Image](https://github.com/user-attachments/assets/0385fa1d-ffea-4197-92e5-963a52c32bc7)
![Image](https://github.com/user-attachments/assets/2f63d239-24a4-4369-95e6-043cf792bef5)
![Image](https://github.com/user-attachments/assets/a514c10f-a6c8-48b0-8392-ab09a3abae1a)
![Image](https://github.com/user-attachments/assets/c07ce2a3-12e2-4742-a717-049d8c1ba04c)
![Image](https://github.com/user-attachments/assets/e5593eac-2bdf-42ed-b8bf-b42ca37fbe24)
![Image](https://github.com/user-attachments/assets/d6b6462f-b291-4e41-be87-e32591ef406b)
![Image](https://github.com/user-attachments/assets/c55ffb25-6869-4789-80ba-cd7ff02e6cb9)
![Image](https://github.com/user-attachments/assets/cdde67ca-63d7-4259-b33c-f5bb8e977498)
![Image](https://github.com/user-attachments/assets/c2246efb-0559-4510-83c9-5c7c82681ee4)
![Image](https://github.com/user-attachments/assets/c7084624-8821-4623-8789-d9887f688462)
![Image](https://github.com/user-attachments/assets/7d9e8122-4cd6-406a-8f36-5612cfd24526)
![Image](https://github.com/user-attachments/assets/25945d93-0063-4388-91b2-eabbd4fabeaf)

### Expected behaviour

Form elements should have a visible label

### How to reproduce

Run axe DevTools
